src/resources/genomicsdb_vid_mapping.proto

```diff
@@ -0,0 +1,36 @@
+/**
+* Note: Variable names are kept aligned with
+* GenomicsDB JSON configuration.
+*
+* To build, use protocol buffer version >3.2.0 and the following command from base directory:
+* $ protoc -Isrc/resources/ --java_out=src/main/java/ src/resources/genomicsdb_vid_mapping.proto
+*/
+
+syntax = "proto2";
+
+option java_package = "com.intel.genomicsdb";
+option java_outer_classname = "GenomicsDBVidMapProto";
+
+message Length {
```

Comment:
If I don't explicitly give a name of the outer class, protobuf compiler sets an arbitrary name which breaks our naming convention (irritating!!)